src/algebra/geom_sum.lean

```diff
@@ -114,6 +146,25 @@ begin
   rw [← this, add_sub_cancel]
 end
 
+lemma commute_op_op_of_commute [has_mul α] {x y : α} (h : commute x y) : commute (op y) (op x) :=
+begin
+  dunfold commute,
+  dunfold semiconj_by,
+  rw [← op_mul, ← op_mul, h.eq]
+end
```

Comment:
https://leanprover-community.github.io/mathlib_docs/algebra/opposites.html might be a reasonable home for this.

Reply:
And `commute.op` might be a convenient name for this so that dot notation works

Reply:
Thanks for the suggestion. I had to add the import to commute but it seemed to have worked out. I generalised to semiconj_by since this was immediate.